Mongo_Assignment/connection_Config.js

```diff
@@ -0,0 +1,33 @@
+const mongoClient = require('mongodb').MongoClient;
+
+let dbObj;
+
+//Declaring environment variable!!!!!
+const username = process.env.USERNAME;
+const password = process.env.PASSWORD;
+
+function connect()
+{
+  //Getting the Database credentails from environment variable!!!
+  console.log(`username is == ${username}`);
+  console.log(`password is ${password}`);
+
+    //Establishing connection with mongoDb!!!!!!
+    mongoClient.connect(`mongodb://${username}:${password}@127.0.0.1:27017`,function(err, db){  
```

Comment:
add mongo url from ENV

Reply:
Added mongo server and port no through env variable